storage/innobase/fsp/fsp0fsp.cc

```diff
@@ -4010,6 +3994,32 @@ dberr_t fil_space_t::garbage_collect(bool shutdown)
     return err;
   }
 
+  /* Remove cached undo log segments in the rollback segment
+  header page exist in system tablespace */
```

Comment:
This is not valid English grammar. At the very least, a word such as "that" or "which" seems to be missing.